src/sentry/static/sentry/app/views/organizationDiscover/result/viewsDropdown.jsx

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import DropdownLink from 'app/components/dropdownLink';
+import MenuItem from 'app/components/menuItem';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import theme from 'app/utils/theme';
+
+class ViewsDropdown extends React.PureComponent {
+  static propTypes = {
+    options: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string,
+        name: PropTypes.string,
+      })
+    ).isRequired,
+    handleChange: PropTypes.func.isRequired,
+    view: PropTypes.string.isRequired,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      view: 'table',
+    };
+  }
+
+  getMenuItem = opt => {
+    return (
+      <MenuItem
+        key={opt.id}
+        onSelect={this.props.handleChange}
+        eventKey={opt.id}
+        isActive={opt.id === this.props.view}
+      >
+        {opt.name}
+      </MenuItem>
+    );
+  };
+
+  render() {
+    const {options} = this.props;
+    let dropdownTitle = <Title>{t('Views')}</Title>;
```

Comment:
The name of the visualization should be present even when the dropdown is closed.

How about making this say for example`View: Line By Day` similar to the dropdown in the stream view. Styling as a button works well there too, shall we copy that here?

Reply:
Hmm, on second thoughts we actually call these "visualizations". View means saved vs query view in the context of discover.

Reply:
Just confirming - do you want it to say `Visualization: Line By Day`? 
And yeah I agree, will change into a button.

Reply:
Yeah, I think that'll work